tests/unit/validation/test_validation_functions.py

```diff
@@ -431,6 +433,34 @@ def test_validate_values__infinite_sigmas(sensor_type, parameter):
         assert not isinstance(error, InfinityError)
 
 
+@pytest.mark.parametrize(
+    ("sensor_type", "parameter"),
+    [
+        ("sym_power_sensor", ["p_sigma", "q_sigma"]),
+        ("asym_power_sensor", ["p_sigma", "q_sigma"]),
+    ],
+)
+def test_validate_values__bad_p_q_sigma(sensor_type, parameter):
+    def random_id(dim=2):
+        return np.random.randint(0, dim)
+
+    def random_fill(array, sensor_type, parameter):
+        if sensor_type == "sym_power_sensor":
+            array[parameter[random_id()]] = np.random.rand()
+        else:
+            array[parameter[random_id()]][0][0] = np.random.rand()
+            array[parameter[random_id()]][0][1] = np.random.rand()
+            array[parameter[random_id()]][0][2] = np.random.rand()
```

Comment:
we tend to not do random testing since what we do is seldom sufficiently generalizable and the same seed may result in different random values (due to differences in built-in random engine, especially in Python) between platforms, making it hard to reproduce.

instead, usually we provide dummy values (based on the `N > 1` argument that if you test `-1, 0, 1, 2` and the implementation is reasonably good, then that is pretty much equivalent to testing `-1, 0, 1, 2, ..., many`).

I do agree that it's not ideal, so i guess we're open to new input, but either fixing and/or clearly logging the `pytest.seed` would be key. also, i know different people have different opinions within the team, so if you want to introduce this, please request some time in the next knowledge sharing session

Reply:
Hmm. Although here the values are randomly set, but following the same erroneous pattern.